rabbitmq/custom_entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+sed 's/RABBITMQ_USER_PLACEHOLDER/'"$RABBITMQ_USER"'/g' /etc/rabbitmq/definitions.json | \
+  sed 's/RABBITMQ_PASS_HASH_PLACEHOLDER/'"$RABBITMQ_PASS_HASH"'/g' >etc/rabbitmq/updated_definitions.json
+rm /etc/rabbitmq/definitions.json
+mv /etc/rabbitmq/updated_definitions.json /etc/rabbitmq/definitions.json
```

Comment:
woulnd't instead of:
```
sed 's/RABBITMQ_USER_PLACEHOLDER/'"$RABBITMQ_USER"'/g' /etc/rabbitmq/definitions.json | \
  sed 's/RABBITMQ_PASS_HASH_PLACEHOLDER/'"$RABBITMQ_PASS_HASH"'/g' >etc/rabbitmq/updated_definitions.json
rm /etc/rabbitmq/definitions.json
mv /etc/rabbitmq/updated_definitions.json /etc/rabbitmq/definitions.json
```
the following also do the job:
```
sed 's/RABBITMQ_USER_PLACEHOLDER/'"$RABBITMQ_USER"'/g' /etc/rabbitmq/definitions.json | \
  sed 's/RABBITMQ_PASS_HASH_PLACEHOLDER/'"$RABBITMQ_PASS_HASH"'/g' >etc/rabbitmq/definitions.json
```

Reply:
WIP

Reply:
I tried that once while writing it and it didn't work. At the time, I did not investigate why it didn't work and will spend an hour or so doing that now. If I can't find a reason fix however I'd leave it as is.

Reply:
Done by using the -i flag of sed command for inplace manipulation.

Reply:
ok